docs/guide/object-backgrounds.md

```diff
@@ -43,24 +43,28 @@ TexturedBackground {
 
 - `texture`: Defines the texture of the background.
 
+- `luminosity`: Is `Background.luminosity`.
+
+- `skybox`: Specifies if the background is used to define the textured skybox.
```

Comment:
I have the feeling that this sentence is not very clear. 
I would use "texture" instead of "background" term because even if the `skybox` field is disabled there is a uniform colored background.

```suggestion
- `skybox`: Specifies if the `texture` field is used to define the skybox.
```

But then I'm also wondering if  `skybox` is a good field name, given that from the description (and `fallbackSkyboxColor` field) it seems that even if this field is set to FALSE a skybox still exists.